### DISCUSSION TITLE
[Account reinstatement] My account has been flagged as spam

### DISCUSSION THREAD

**Discussion by luoabner** [community/community#122270] [New to GitHub] [answered]
Hey @canuckjacq @davecheney,

Sorry to bother you. But I have come to difficulties. My account has been marked as spam, and I send a reinstatement to GitHub on 2 April. Till now, there's no responds. Could you please help me to get it recovered? Thanks!!!

I'm sending this discussion with another account, because I find no matter what I do in my previous account, anybody else can not see.

This is the original support ticket:

Does your claim involve content on GitHub or npm.js?

GitHub

What is the username and repository or package name that was impacted?

abnerhexu

Why are you requesting reinstatement? If you are requesting support for your account not related to moderation limits made on your account by GitHub please raise a support ticket instead.

I can login, but my profile and contributions aren't visible to others

Have you previously contacted GitHub about this claim?

No

Would you like to provide any additional information or context that would be helpful for our review of your reinstatement request?

Hello, my GitHub account has been marked as spam, and I am not sure how this happened. My computer may have been subjected to a virus attack because a few days ago when I logged into my GitHub account, I found a repository named adobe-animated, which I never created, so I deleted it. I have strengthened the security of my account, including enabling 2FA and GitHub Mobile verification. I am very anxious now because I need to use GitHub to complete my "Compiler Principles" assignment, and this repository is located at https://github.com/abnerhexu/SysY.

Furthermore, I just discovered another repository on my GitHub account that I never created, with the address https://github.com/abnerhexu/abnerhexu1. I am now preparing to delete it.

I hope you can help me restore my account as soon as possible. Thank you!

I have reviewed and understand the GitHub Acceptable Use Policies and Community Guidelines.

**Comment by Anish570** [comment]
Certainly! Here's a concise, point-wise response addressing your issue:

1. **Resend Your Reinstatement Request**: It's unfortunate that you haven't received a response yet. Given the importance of this matter to you, I recommend resending your reinstatement request to GitHub. Sometimes, requests may get lost or overlooked, so sending it again can ensure it gets attention.

2. **Include Additional Details**: In your new email, include any additional information or context that could help expedite the process. Mention the steps you've taken to secure your account, such as enabling 2FA and GitHub Mobile verification. This demonstrates your commitment to account security and may aid in your reinstatement.

3. **Provide Evidence of Unauthorized Activity**: Emphasize the unauthorized activity on your account, such as the creation of repositories you didn't make, like "adobe-animated" and "abnerhexu1." This evidence strengthens your case and highlights that your account may have been compromised.

4. **Highlight Urgency**: Clearly communicate the urgency of your situation, especially if you have pending assignments or projects reliant on GitHub access. This can encourage GitHub support to prioritize your request.

5. **Express Compliance with Policies**: Reiterate your compliance with GitHub's Acceptable Use Policies and Community Guidelines. This reaffirms your commitment to maintaining a positive presence on the platform.

6. **Follow Up**: After sending your reinstatement request, consider following up with GitHub if you haven't received a response within a reasonable timeframe. This shows persistence and ensures your request doesn't slip through the cracks.

By following these steps and presenting your case clearly and concisely, you increase the likelihood of a prompt and favorable response from GitHub support. Best of luck in resolving this issue swiftly!

**Reply by luoabner** [reply]
Wow... Thanks for your kindly advice. But are you a human... or a GPT-like robot?

**Reply by Anish570** [reply]
LOL 😂

**Reply by nicolasZolet** [reply]
Hey @luoabner, 

I have the same issue, any update from the github?

As you, I have create a ticket with this problem weeks ago, but then nobody reply me.

**Comment by Wwright91** [comment]
I have the same problem. Its been 3 weeks for you with no resolve and going on 2 weeks for me. This needs to be addressed swiftly

**Reply by HimanshuMohanty-Git24** [reply]
same here @Wwright91 there is no response till now from there side at least they should give reason why the account was flagged.
![image](https://github.com/community/community/assets/94133298/be3deab2-8ac9-40cc-b277-4f60f3ac72cd)

**Comment by wuxin0011** [comment]
I also encountered the same problem. How did you recover it

**Comment by wuxin0011** [comment]
I clicked on the connection direction. Your account is still 404. Has the official replied to any messages from you

**Comment by piotryordanov** [comment]
Yeah so they block you in a blink of an eye, then can make you wait forever to get a response. Mine has been pending review for over 45 days. Thank you github for the horrible customer experience

**Comment by Sahil-shaikh-dev** [comment]
I have the same issue and my ticket number is  #2980633.
It's been 2 weeks and I didn't get any response from Git Hub also, the first time when I submitted the ticket it automatically got closed then I needed to reopen it.

I can not host my portfolio because of this and I cannot show my projects or profile to anyone and my interviews are coming up.

**Comment by sahil-shekh** [comment]
I have the same issue and my ticket number is #2980633. this is my other account my account where I am facing this issue is https://github.com/Sahil-shaikh-dev. I cannot see the comment which I have made from that account.
It's been 2 weeks and I didn't get any response from Git Hub also, the first time when I submitted the ticket it automatically got closed then I needed to reopen it.

I can not host my portfolio because of this and I cannot show my projects or profile to anyone and my interviews are coming up.

**Reply by AHMED-salah00** [reply]
Sorry for that. I am having the same problem here 😭

**Comment by rushhiii** [comment]
Hey @luoabner, sorry you're going through this — it’s definitely frustrating.

A few things that can help speed things up:

🔁 **Submit a follow-up to your support ticket** via [GitHub Support](https://support.github.com/contact). Even if you’ve already submitted once, a respectful follow-up explaining the urgency (e.g. coursework deadlines) often gets faster traction.

🔐 **Mention your security improvements** (2FA, GitHub Mobile auth, repo deletions) — this shows you’re proactive about safety, and that the suspicious activity wasn’t intentional.

🧾 **Include links to impacted repos** (like your SysY compiler assignment) in the message — this helps support staff verify you’re a legit dev, not a bot/spammer.

⏳ Unfortunately, flagged accounts are invisible to others and often take a while to review manually. You can still use Discussions like you’re doing here — that’s a smart move.

I hope this gets resolved quickly — I’ve seen others get reinstated after following up with this kind of detail. Good luck!

— @rushhiii

**Comment by MaskedBug601** [comment]
I have the same issue could youu please help out me as well this is my ticked number #4550708

**Reply by abnerhexu** [reply, marked as answer]
You can just use another account and write the reinstatement request. Hope you good luck.

**Reply by MaskedBug601** [reply]
Thanks a lot got it back.

**Reply by sebgam2** [reply]
Hi! I saw that you were able to get your suspended account restored. Would you mind sharing what you did? Did GitHub eventually respond to your support ticket, or was there another way you contacted them? I've been waiting over 15 days with two open tickets and no response. Any advice would be greatly appreciated. Thank you!

**Reply by luoabner** [reply]
> Hi! I saw that you were able to get your suspended account restored. Would you mind sharing what you did? Did GitHub eventually respond to your support ticket, or was there another way you contacted them? I've been waiting over 15 days with two open tickets and no response. Any advice would be greatly appreciated. Thank you!

Please see https://github.com/orgs/community/discussions/122270#discussioncomment-17610386

**Comment by Willamson23** [comment]
If you've already submitted a reinstatement request, there's unfortunately not much else you can do except wait for GitHub Support to review it. Since you've already secured your account with 2FA and removed the suspicious repositories, you've taken the right steps. I hope your account gets restored soon.

**Comment by sebgam2** [comment]
Hello,

My GitHub account has been suspended for over 15 days. I have two open support tickets (#4563050 and #4590235) but have not received any response.

This account manages the production repositories for our software company, and the lack of communication is having a critical impact on our business.

Could a GitHub team member please review my support tickets or advise on the next steps?

Thank you very much.

**Reply by luoabner** [reply]
see https://github.com/orgs/community/discussions/122270#discussioncomment-17610386
